emamux.el

```diff
@@ -424,6 +424,20 @@ For helm completion use either `normal' or `helm' and turn on `helm-mode'."
   (emamux:check-runner-alive)
   (emamux:tmux-run-command nil "resize-pane" "-Z" "-t" emamux:runner-pane-id))
 
+(defvar emamux:keymap
+  (let ((map (make-sparse-keymap)))
+    (define-key map "\C-s" #'emamux:send-command)
+    (define-key map "\C-y" #'emamux:yank-from-list-buffers)
+    (when (emamux:in-tmux-p)
+      (define-key map "\C-r" #'emamux:run-command)
+      (define-key map "\M-r" #'emamux:run-last-command)
```

Comment:
I propose using `M-!` as the `run-command`, which is analogous with plain `M-!` (`shell-command`), but then I assume we bump into the problem of "what to do with the run-last-command?"

For me, `M-!` and then `C-r` would give me a history, so binding the run-last-command was not so much of my interest.

This is just my opinion so I don't mind if you just ignored this comment and merged your PR.